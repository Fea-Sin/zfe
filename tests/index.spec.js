import React from 'react';
import { mount, render } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';
import Comp from '../src';

describe('zfe', () => {
  it('base test comp', () => {
    const wrapper = mount(
      <Comp />
    )
    expect('COMP IS DONE').toBe('COMP IS DONE')
  })
  it('snapshot to match', () => {
    const wrapper = render(
      <Comp />
    )
    expect( renderToJson(wrapper) ).toMatchSnapshot()
  })
})
