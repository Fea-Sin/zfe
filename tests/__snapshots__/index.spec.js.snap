// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`zfe snapshot to match 1`] = `
<div
  class="cr-app"
>
  <div>
    echarts version 4.6.0
  </div>
  <div>
    ZFG version 1.0.2
  </div>
</div>
`;
